lib/samsam.js

```diff
@@ -1,17 +1,19 @@
 "use strict";
 
-var deepEqualCyclic = require("./deep-equal");
 var identical = require("./identical");
 var isArguments = require("./is-arguments");
 var isElement = require("./is-element");
 var isNegZero = require("./is-neg-zero");
 var match = require("./match");
+var deepEqualCyclic = require("./deep-equal").use(match);
+var createMatcher = require("./sinon-match");
```

Comment:
These were previously in alphabetical order.

Reply:
Yes, but `deepEqualCyclic` needs `match` to be defined.